library/bigdecimal/new_spec.rb

```diff
@@ -4,102 +4,102 @@
 describe "BigDecimal.new" do
```

Comment:
This should be changed to `"Kernel#BigDecimal"` then.
And the file should be moved to `library/bigdecimal/BigDecimal_spec.rb`.